.github/release_template.md

```diff
@@ -0,0 +1,30 @@
+<!--
+This is the template for new release issues.
+-->
+
+AC:
+
+ - [ ] [GIT] Create branch release_0.9.16 in owncloud/android-library from master  (almost feature-frozen)
+ - [ ] [GIT] Create branch release_2.3.0 in owncloud/android from master (almost feature-frozen)
+ - [ ] [DEV] Update version number and name in AndroidManifest.xml in android module
+ - [ ] [DEV] Update version number and name in AndroidManifest.xml in oc_jb_workaround module
+ - [ ] [DIS] Create changelog file (< 500 chars) and add to CHANGELOG.md in owncloud/android
+ - [ ] [QA] Design Test plan  @jesmrec
+ - [ ] [QA] Regression Test plan @jesmrec
+ - [ ] [GIT] Merge branch release_0.9.16 in owncloud/android-library, into master
+ - [ ] [GIT] Merge branch release_2.3.0 in owncloud/android, into master
+ - [ ] [DIS] Generate test APKs and BAR files from branch master in owncloud/android
+ - [ ] [GIT] Create and sign tag oc-android-2.3.0 in HEAD commit of stable master branch, in owncloud/android
+ - [ ] [GIT] Create and sign tag oc-android-library-0.9.16 in HEAD commit of stable master branch, in owncloud/android-library
```

Comment:
In the client repo, everything is generic or an example:
https://github.com/owncloud/client/blob/organize-templates/.github/release_template.md